### DISCUSSION TITLE
Use as realtime Radon sniffer

### DISCUSSION THREAD

**Discussion by click3dm** [ozel/DIY_particle_detector#54] [Ideas]
Hello everyone,

I was very happy discovering this cool device for alpha detection. 
What I am curious about: Does anyone have experience in using this as a "real-time" radon sniffer?
So I would imagine a little fan, that breaths in the ambient air continuously and after some seconds / minutes of measuring, you get a first indication of radon levels, in order to detect leaks or similar in buildings.
A relative comparison would already be quite nice (to compare critical pathways inside buildings e.g. with outside air); absolute scale quite better.
There are commercially available tools for sure, but financially out of league.

Thanks for open discussion!

**Comment by ozel** [comment]
it's absolutely possible, see my comments here: https://github.com/ozel/DIY_particle_detector/discussions/51#discussion-4977492
If you get anywhere let me know, I'd join & support. I also pitched this idea once based on the alpha-spectrometer version but had no time yet to follow through.
The hard stuff is getting the collection camber right. Some electrostatic field helps but needs careful adjustment together with air flow. A passive diffusion-based chamber with air vents (= do nothing, wait for alpha pulses from polonium and count) would be the simplest approach. 
This is could be a nice starting point: https://spectrum.ieee.org/how-to-build-a-homebrew-radon-detector. Replace the webcam with the alpha spectrometer/BPX61.
